UIInfoSuite2/ModEntry.cs

```diff
@@ -160,16 +161,36 @@ private static void HandleCalendarAndQuestKeyBindings(IModHelper helper)
   {
     if (_modConfig != null)
     {
-      if (Context.IsPlayerFree && _modConfig.OpenCalendarKeybind.JustPressed())
+      if (_modConfig.OpenCalendarKeybind.JustPressed())
       {
         helper.Input.SuppressActiveKeybinds(_modConfig.OpenCalendarKeybind);
-        Game1.activeClickableMenu = new Billboard();
+        ToggleBillboard(false);
       }
-      else if (Context.IsPlayerFree && _modConfig.OpenQuestBoardKeybind.JustPressed())
+      else if (_modConfig.OpenQuestBoardKeybind.JustPressed())
       {
         helper.Input.SuppressActiveKeybinds(_modConfig.OpenQuestBoardKeybind);
-        Game1.RefreshQuestOfTheDay();
```

Comment:
I don't think this is necessary since the daily quest is refreshed every morning. Please let me know if I'm missing something.